lib/neo4j-core/query_clauses.rb

```diff
@@ -727,6 +728,75 @@ def clause_join
           end
         end
       end
+
+      class UnionClause < Clause
+        KEYWORD = 'UNION'
+
+        # If `value` is a query object, returns value.to_cypher. Formatting optional
+        def from_query(value, pretty: false)
+          from_string(value.to_cypher(pretty: pretty))
```

Comment:
I don't remember seeing other uses of KW args in the neo4j-core source. I'm not sure if this was personal preference or if you're targeting a version of Ruby pre-KW args

Reply:
You're right that they aren't used much.  In the past we've supported Ruby 1.9, but we require at least Ruby 2.1 now, so keyword arguments should be fine.